x-pack/legacy/plugins/searchprofiler/public/app.js

```diff
@@ -143,7 +147,7 @@ function profileVizController($scope, $http, HighlightService) {
       }
 
       $scope.renderProfile(resp.data.resp.profile.shards);
-    }).catch(notify.error);
+    }).catch(reason => toastNotifications.addDanger(reason));
```

Comment:
The `reason` shape is not what `addDanger` expects, so this causes an error and renders and empty toast. We can fix this by importing this helper:

```js
import { formatAngularHttpError } from 'ui/notify/lib';
```

And then changing this line to consume it:

```js
toastNotifications.addDanger(formatAngularHttpError(reason))
```

![image](https://user-images.githubusercontent.com/1238659/60992001-7af95980-a300-11e9-91fd-e225784f524b.png)